### DISCUSSION TITLE
Alternative to ConvertImageCommand() in MagickWand/deprecate.c

### DISCUSSION THREAD

**Discussion by ptiruvaipati** [ImageMagick/ImageMagick#7391] [Help] [answered]
### ImageMagick version

7.1.1-33

### Operating system, version and so on

Windows 10

### Description

Hi Team,

We are upgrading ImageMagick from 7.1.1-11 to 7.1.1-33. 

We have a dependency with **ConvertImageCommand()** method in  **convert.c**(MagickWand) and 

this method is deprecated in 7.1.1-33. 

Can you please tell me alternative to ConvertImageCommand() method?

Regards,
Pavan

**Comment by snibgo** [comment, marked as answer]
I suggest function `MagickImageCommand()` which is in `MagickWand/magick-cli.c`.

**Reply by ptiruvaipati** [reply]
Thank you for the quick help.
